example_project/measures/BuildResidentialModel/measure.rb

```diff
@@ -325,21 +326,43 @@ def get_unit_positions(runner, args)
 
         geometry_unit_left_wall_is_adiabatic = true
         geometry_unit_right_wall_is_adiabatic = true
-        geometry_unit_front_wall_is_adiabatic = true
-        geometry_unit_back_wall_is_adiabatic = false
+        geometry_unit_front_wall_is_adiabatic = false
+        geometry_unit_back_wall_is_adiabatic = true
```

Comment:
Switching these because the docs say the low-rise multifamily building is double _exterior_ corridor.